test/blockdiagonal.jl

```diff
@@ -90,4 +90,13 @@ using Test
         @test Matrix(b1) ≈ b1
         @test b1 ≈ Matrix(b1)
     end  # Equality
+
+    @testset "Non-Square Matrix" begin
+		A1 = ones(2, 4)
+		A2 = 2 * ones(3, 2)
+		B1 = BlockDiagonal([A1, A2])
+        B2 = [A1 zeros(2, 2); zeros(3, 4) A2]
+
+		@test B1 == B2
+    end  # Non-Square Matrix
```

Comment:
The whitespace is still off in thies block

Reply:
Sorry, I don't get which white space you are talking about. Could you edit it the way it is your standard?